tools/nuget/generate_nuspec_for_native_nuget.py

```diff
@@ -74,7 +74,7 @@ def generate_license(line_list):
     line_list.append('<license type="file">LICENSE</license>')
 
 def generate_readme(line_list):
-    line_list.append('<readme>README.md</readme>')
+    line_list.append('<readme>nuget/PACKAGE.md</readme>')
```

Comment:
README is inside src\csharp:

https://github.com/microsoft/onnxruntime-genai/blob/main/src/csharp/README.md

Reply:
I moved it